mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -973,6 +978,7 @@ func ResourceComposerEnvironment() *schema.Resource {
 						"gke_cluster": {
 							Type:        schema.TypeString,
 							Computed:    true,
+							Optional:    true,
```

Comment:
This likely should just stay Computed. Adding optional indicates that it is settable but has some kind of server-side default, but [go/composer-3-terraform](http://go/composer-3-terraform) I believe the intended behavior here is to support it not being returned at all (which is different.)

Reply:
Right, this was the reason for adding Computed. I removed it.